ci-operator/step-registry/interop-tests/openshift-virtualization-tests/interop-tests-openshift-virtualization-tests-commands.sh

```
@@ -1,231 +1,262 @@
-
-    # Use file flag so loop can be interrupted by removing the file
-    touch /tmp/debug_marker
-    while [[ -f /tmp/debug_marker ]]; do
-        sleep 120
-    done
-  fi
+DebugOnExit() {
+    typeset -i exitCode=$?
+    typeset -i endTime=$SECONDS
+    typeset -i executionTime=$((endTime - startTime))
+    typeset -i debugThreshold=720 # 12 minutes in seconds
```

Shellcheck (0.11.0)

[warning] debugThreshold appears unused. Verify use (or export if used externally).

(SC2034)